farm/modeling/prediction_head.py

```diff
@@ -1200,12 +1202,19 @@ def formatted_preds(self, logits=None, preds=None, baskets=None, **kwargs):
         (see Inferencer._get_predictions_and_aggregate()).
         """
 
+        # Log cases where this method receives the wrong kind of input
+        logit_input = logits is not None
+        preds_input = preds is not None
+
+        if logit_input:
+            logger.warning("QuestionAnsweringHead.formatted_preds() received logit input when it only expects pred input")
+        if not preds_input:
+            logger.warning("QuestionAnsweringHead.formatted_preds() did not receive the preds input it expects")
```

Comment:
this should be at least an logger.error or even kept as assert, since the app breaks without preds